core/src/Lender.sol

```diff
@@ -208,12 +208,14 @@ contract Lender is Ledger {
         // Accrue interest and update reserves
         (Cache memory cache, ) = _load();
 
-        // Convert `amount` to `units`
-        units = amount.mulDivUp(BORROWS_SCALER, cache.borrowIndex);
+        unchecked {
+            // Convert `amount` to `units`
+            units = (amount * BORROWS_SCALER) / cache.borrowIndex;
 
-        // Track borrows
+            // Track borrows
+            borrows[msg.sender] = b + units;
+        }
         cache.borrowBase += units;
-        borrows[msg.sender] = b + units;
```

Comment:
https://github.com/aloelabs/aloe-ii/issues/58

Changing the function anyway at this point, figured I might as well go back and do this.